src/likes.js

```diff
@@ -0,0 +1,15 @@
+const baseUrl = 'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/Jxa2o7XJAhJoB2RZ7ghK/likes';
+// const id = Jxa2o7XJAhJoB2RZ7ghK;
+const addLikes = (id) => fetch(baseUrl, {
+  method: 'POST',
```

Comment:
Please remove commented code from `likes.js` and from all other files wherever present as it can mess up the whole code base.

Reply:
Removed.